pinot-core/src/main/java/org/apache/pinot/core/data/manager/realtime/IdleTimer.java

```diff
@@ -0,0 +1,85 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.pinot.core.data.manager.realtime;
+
+/**
+ * The IdleTimer is responsible for keeping track of 2 different idle times:
+ *  - The stream idle time which resets every time we remake the stream consumer.
```

Comment:
I don't fully follow the difference between these 2 idle time. Currently seems we always update and reset them together

Reply:
I redeployed this internally, and I'm seeing the log volume come down a lot. Thanks for catching!

Reply:
IIUC, steam idle means consumer idle (reset when we recreate consumer), consume idle means no event (to me that is closer to stream idle). Should we consider renaming them? IMHO current names are a little bit confusing